ext/openssl/openssl.stub.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+function openssl_x509_export_to_file($x509, string $outfilename, bool $notext = true): bool {}
+
+function openssl_x509_export($x509 , string &$out, bool $notext = true): bool {}
```

Comment:
`&$out` should not have a type: For reference parameters the type is still the input type (which is unrestricted), while there is no way to specify the output type (string).

Reply:
Updated